application/controllers/last/Welcome.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/**
+ * Description of Last
+ *
+ * @author yoseph
+ */
+class Welcome extends Application {
+
+	function __construct()
+	{
+		parent::__construct();
+	}
+
+	//-------------------------------------------------------------
+	//  The normal pages
+	//-------------------------------------------------------------
+
+	function index()
+	{
+		$this->data['pagebody'] = 'justone';	// this is the view we want shown     
+                
+                $record = $this->quotes->last();
+                $this->data = array_merge($this->data, $record);
```

Comment:
lines 29 and 30 - why the extra space beforehand?  should be in line with the rest.